dev-python/nuitka/nuitka-0.6.14.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS="rdepend"
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1 flag-o-matic optfeature
+
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="https://www.nuitka.net"
+SRC_URI="https://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-util/scons[${PYTHON_USEDEP}]"
```

Comment:
Definitely?

Reply:
I am not sure what you mean here ...

Reply:
Scons, being a build system, is a weird runtime dep

Reply:
It's normal here.
```
$ nuitka3 --help | grep show-scons
    --show-scons        Operate Scons in non-quiet mode, showing the executed
```

Reply:
Sure, but having no knowledge of it, it was a reasonable question to ask :)

Reply:
Ah, OK. Nuitka is a transpiler compier . It first transpiles the Python code to C, and then compiles it to native code. For the second part it uses scons.